frontend/src/UseFormSender.js

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+import axios from 'axios';
+
+const UseFormSender = (reqType, url) => {
+    const [error, setError] = useState(null);
+    const [loading, setLoading] = useState(false);
+    const [data, setData] = useState(null);
+
+    const cb = async (requestBody) => {
+        setLoading(true);
+        setError(null);
+        try {
```

Comment:
let your code breath, please add a new line before try and after catch's closing bracket.